src/custom-types.json

```diff
@@ -50,6 +50,20 @@
       "http://www.iana.org/assignments/media-types/application/geo+json"
     ]
   },
+  "application/grpc": {
+    "sources": [
+      "https://tools.ietf.org/html/draft-kumar-rtgwg-grpc-protocol-00",
```

Comment:
This draft expired January 9, 2017. Once it is expired it is obsolete. Was the draft ever renewed or accepted? Please change the link to at least something not currently expired 👍